### ISSUE TITLE
Translate <br> javadoc tags

### ISSUE THREAD

**Issue by pumpkinlink** [vojtechhabarta/typescript-generator#309]
My Javadoc comments have a lot of ```<br>``` tags in order to correctly parse line-breaks on Java IDEs (Eclipse, IntelliJ). The problem is, apparently JSDoc only supports markdown-style linebreaks (at least on VSCode). 

Is there a way to translate ```<br>``` tags to double line breaks? (while also ignoring actual line breaks, in order to avoid potential triple line breaks)

#### Example:
* My generated typescript definiton:
``` typescript
/**
 * DOCUMENTAÇÃO DA CLASSE <br>
 * ---------------------- <br>
 * FINALIDADE: <br>
 * TODO Definir documentação da classe. <br>
 * <br>
 * HISTÓRICO DE DESENVOLVIMENTO: <br>
 * Oct 23, 2018 - @author lucasisrael - Primeira versão da classe. <br>
 * <br>
 * <br>
 * LISTA DE CLASSES INTERNAS: <br>
 */
export interface RDomainObject extends RMacadameClass {
}
```
* How it shows up inside VSCode hover pop-up:
![l2zii1f](https://user-images.githubusercontent.com/11864620/52669873-65713680-2efe-11e9-9dc6-195e4ba6d76d.png)

* How it should be:
![b8bqeor](https://user-images.githubusercontent.com/11864620/52669491-51790500-2efd-11e9-91a7-bea28bdd97ea.png)

**Comment by vojtechhabarta** [comment]
The problem is in general that Javadoc uses HTML while JSDoc usually uses Markdown. So what about other HTML tags? For example HTML tables can be embedded inside Markdown so we probably don't want to process or remove all HTML tags.

Maybe just handle `<br>` is right thing to do. Do you think it would work for all or do we need configuration switch for it?

**Comment by pumpkinlink** [comment]
You're right, besides the br tags, there is seemingly also other ones commonly used inside javadoc like ```pre```,  ```ul```/```li``` and ```strong```.  But I think line breaks are a priority, to at least make the text more readable.

Also apparently some tools use ```p``` instead of ```br``` for line breaks.

In regard to the configuration switch I think it would be the best to do have it. As JSDoc3 spec does not officially support any markup outside of ```@``` tags, it's just that Markdown is supported by editors (both VS and VSCode do, and I suspect that VSCode is one of the most used editors for typescript projects)

**Comment by vojtechhabarta** [comment]
I pushed commit which replaces `<br>`, `<p>` and `</p>` with double line break.
Could you please try it (from `master` branch) or at least review this commit?

**Comment by pumpkinlink** [comment]
> I pushed commit which replaces `<br>`, `<p>` and `</p>` with double line break.
> Could you please try it (from `master` branch) or at least review this commit?

Tested on my project and working perfectly. Thank you very much for the great work!

I would add `<br/>` variations though, for xhtml-valid version of the `br` tag. So I'm creating a pull request for that.

I'm not sure if the tests are right or if I should add a specific test for each variation. All tests are passing though.

**Comment by vojtechhabarta** [comment]
Thanks for trying the change and for your improvement.
I merged your PR, tests are ok.

**Comment by vojtechhabarta** [comment]
I released [v2.13.489](https://github.com/vojtechhabarta/typescript-generator/releases/tag/v2.13.489) with this feature.
